File: examples/Yive/ServerStatus.pl
#!/usr/bin/perl
use strict;
use HTML::Template;
use LWP::Simple qw(get);
use JSON qw( decode_json );
my $decoded = decode_json(get("http://mcapi.ca/query/us.mineplex.com/info"));
print qq(Content-type: text/html\n\n);

# Status: $info->{'status'}
# MOTD: $info->{'motd'}
# Version: $info->{'version'}
# Players Online: $info->{'players'}{'online'}
# Players Max: $info->{'players'}{'max'}
# Ping: $info->{'ping'}

# This will not display the player names who are online, you'll need to use the list API.
if(!$decoded->{'status'}) {
	print 'The server is offline';
} else {
	print 'The server is online with '.$decoded->{'players'}{'online'}.'/'.$decoded->{'players'}{'max'}.' players online.'
}

File: examples/Yive/ServerStatus.php
<?php
$serverip = "localhost";
$info = json_decode( file_get_contents( 'http://mcapi.ca/query/'.$serverip.'/info' ), true ); 

// Status: $info['status']
// MOTD: $info['motd']
// Version: $info['version']
// Players Online: $info['players']['online']
// Players Max: $info['players']['max']
// Ping: $info['ping']
//
// This will not display the player names who are online, you'll need to use the list API.
if(!$info['status']) {
	echo 'The server is offline';
} else {
	echo 'The server is online with '.$info['players']['online'].'/'.$info['players']['max'].' players online';
}
?>